src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.Designer.targets

```diff
@@ -38,6 +38,9 @@ Copyright (C) 2016 Xamarin. All rights reserved.
     <ExtraJarLocation Include="@(_LibraryJars)">
       <Source>LibraryImport</Source>
     </ExtraJarLocation>
+    <ExtraJarLocation Include="@(AndroidJavaLibrary)">
```

Comment:
It looks like we never documented `%(ExtraJarLocation.Source)`, much less `@(ExtraJarLocation)`.

These don't (necessarily) need public documentation, but they *at least* need to be documented in a commit message!  This was apparently overlooked in commit fe0ae91fd.

xamarin/monodroid@d825c78a appears to be the "originator" of `%(ExtraJarLocation.Source)`, but that *likewise* doesn't document the semantics of the `@(ExtraJarLocation)` item group or the `%(ExtraJarLocation.Source)` metadata.

Are there any docs for these constructs?  What do they *mean*?  What does it mean if `%(ExtraJarLocation.Source)` is `AndroidJavaLibrary`?

Is including `@(AndroidJavaLibrary)` *enough*? What about [`@(AndroidAarLibrary)`](https://github.com/xamarin/xamarin-android/blob/7a9204da99136d8209037fbe34be29feb4ca2dc1/Documentation/guides/BuildProcess.md#androidaarlibrary)?